.eslintrc.json

```diff
@@ -35,14 +35,13 @@
         "allowExpressions": true
       }
     ],
-    "@typescript-eslint/interface-name-prefix": ["error", "always"],
     "@typescript-eslint/array-type": [
       "error",
       {
         "default": "array-simple"
       }
     ],
-    "@typescript-eslint/ban-ts-ignore": "warn",
+    "@typescript-eslint/ban-ts-ignore": "off",
```

Comment:
Any reason for that? But i agree that this is not a cool lint. Always adding two comments for `ts-ignore` is annoying...

Reply:
In case of `"warn"` the linter throws an error which I could not fix otherwise.

>  1:1  error  Definition for rule '@typescript-eslint/ban-ts-ignore' was not found  @typescript-eslint/ban-ts-ignore